### PR TITLE
Versioned.sol inheritance

### PR DESCRIPTION
**Pull request by bulbozaur**

### CODE REVIEW

File: contracts/0.4.24/utils/Versioned.sol [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    /// @notice Returns the current contract version.
+    function getContractVersion() public view returns (uint256) {
+        return CONTRACT_VERSION_POSITION.getStorageUint256();
+    }
+
+    function _checkContractVersion(uint256 version) internal view {
+        require(version == getContractVersion(), "UNEXPECTED_CONTRACT_VERSION");
+    }
+
+    function _writeContractVersion(uint256 version) internal {
```
Comment (arwer13) [inline contracts/0.4.24/utils/Versioned.sol:42]: Maybe `_setContractVersion`? Because we use set but write everywhere else

**Review by arwer13 — CHANGES_REQUESTED** [review]

File: contracts/0.4.24/utils/Versioned.sol (hunk 1 above)
Comment (TheDZhon) [reply contracts/0.4.24/utils/Versioned.sol:42]: agree

Comment (bulbozaur) [reply contracts/0.4.24/utils/Versioned.sol]: fixed

**Review by TheDZhon — APPROVED** [review]
Awesome, thanks! :rocket:

**Pull request merged**